index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width">
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous"><link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+  
+</head>
+<body>
+
+  <header>
+    <appbar class="appbar">
+        <ul class="appbar status">
+            <li>
+                <img src="sources/Time Style.svg"></img>
+            </li>
+            <li>
+                <img src="sources/Group 3.svg"></img>
+            </li>
+        </ul>
```

Comment:
- [ ] Great job with your design 👏. This part of the Figma design shouldn't be part of your design. Kindly remove it 🙏 

![image](https://user-images.githubusercontent.com/74814780/205413723-3239002e-0b7d-4b15-91fa-e8a9217888a9.png)